src/App/Fossa/Config/Common.hs

```diff
@@ -482,6 +497,14 @@ commonOpts =
           , boldItalicized "Default: " <> coloredBoldItalicized Green "full"
           ]
 
+configHelp :: Maybe (Doc AnsiStyle)
```

Comment:
I don't see a reason for these to be `Maybe` since it doesn't seem like there's any possibility that they will be `Nothing`. This forces consumers of that data to handle the case of `Nothing` regardless though.

Reply:
These help docs are of type `Maybe` due to [Options.Applicative](https://hackage.haskell.org/package/optparse-applicative-0.18.1.0/docs/Options-Applicative.html) `helpDoc` function. It required a`Maybe` type.